### ISSUE TITLE
CJ4: Rudder/nose wheel input during take-off

### ISSUE THREAD

**Issue by kutak11** [Working-Title-MSFS-Mods/fspackages#1021]
I am not sure if it is just my bad technique but there is lack of rudder input during take off roll when crosswind. When I am approaching Vr speed the rudder just stop working and I am blowed away to the side of the runway. Never experienced on other types.
May be possible separate nose wheel steering and rudder input... I dont know

Thanks

**Comment by Ayotteted123** [comment]
Look at your sensitivity settings and try resetting them

**Comment by K20017** [comment]
Stems from a flight model issue.  Hopefully tweaks and improvements can be made in the future.

**Comment by Teneus** [comment]
Yeah, added the mod yesterday and it already happened to me on 1 take-off and 2 landings x') Hoping they'll increase the yaw sensitivity !

**Comment by neilhewitt** [comment]
This is still open and unaddressed 7 months after the last comment and well over a year since it was created. This is a significant issue for all users of this aircraft in any kind of crosswind takeoff or landing. Surely this deserves attention from the team? I just had to abandon a flight because I ran off the runway on takeoff in a crosswind because I had insufficient rudder authority. A CJ4 can handle a 14 knot crosswind in RL!

**Comment by mattnischan** [comment]
@neilhewitt 
This is an open source project that was done by volunteers in their free time. As such, anyone in the community is free to submit a fix for this issue if they like.

As we have mentioned in multiple locations, the mod has been in compatibility mode fixes only from the WT team proper since March of last year. There will be no other fixes from us other than for sim update compatibility unless submitted to us by the community.

The CJ4 refresh, due this year sometime, will use a different flight model which should not exhibit this issue.

**Comment by neilhewitt** [comment]
@mattnischan I wasn't aware that the mod was parked pending a refresh, so apologies if I came across as entitled, re-reading my comment I can see how it would seem like that. And of course I know WT is a volunteer group and I 100% appreciate all the work you've all done for our community. 

If I had the necessary skills to try to fix this and submit a PR, I absolutely would. Sadly while I am a developer I'm not a flight sim developer (at least not in terms of creating aircraft models) and I don't have the experience tweaking MSFS flight models to even know where to start.
